### ISSUE TITLE
Add Execute Feedback and Feedback start/stop

### ISSUE THREAD

**Issue by lumnt** [bitfocus/companion-module-chamsys-quickq#17]
Hi,

Before this module existed, we used a custom OSC module to control the QuickQ and Chataigne to translate the feedback from the QuickQ to companion sent to custom variables.

Execute feedback are possible once the feedback is activated. Could these be added to the module? Here's how they are received:

/exec/1/1 : 0 
/exec/1/1 : 1

The last number after the / is the execute number, example execute 5 would be /exec/1/5
The very last number is the state, 1 for on, 0 for off.

Thank you!

**Comment by pepsijon** [comment]
Has this been added to a new version?  I'm not seeing it in the stable or beta versions of Companion.

**Comment by lumnt** [comment]
Nope! We submitted a pull request, but no maintainers have responded nor merged it. Not sure how to notify them.
